apps/frontend/src/modules/notifications/slices/notification.slice.ts

```diff
@@ -38,6 +44,40 @@ const { actions, name, reducer } = createSlice({
 			state.notificationsTotalCount = initialState.notificationsTotalCount;
 			state.dataStatus = DataStatus.REJECTED;
 		});
+
+		builder.addCase(loadAllUnread.pending, (state) => {
+			state.unreadDataStatus = DataStatus.PENDING;
+		});
+		builder.addCase(loadAllUnread.fulfilled, (state, action) => {
+			state.unreadNotifications = action.payload.items;
+			state.unreadDataStatus = DataStatus.FULFILLED;
+		});
+		builder.addCase(loadAllUnread.rejected, (state) => {
+			state.unreadNotifications = [];
+			state.unreadDataStatus = DataStatus.REJECTED;
+		});
+
+		builder.addCase(markAsRead.pending, (state) => {
+			state.markAsReadStatus = DataStatus.PENDING;
+		});
+		builder.addCase(markAsRead.fulfilled, (state, action) => {
+			const { notificationIds } = action.meta.arg;
+
+			state.notifications = state.notifications.map((notification) =>
+				notificationIds.includes(notification.id)
+					? { ...notification, isRead: true }
+					: notification,
+			);
+
+			state.unreadNotifications = state.unreadNotifications.filter(
+				(notification) => !notificationIds.includes(notification.id),
+			);
+
+			state.markAsReadStatus = DataStatus.FULFILLED;
```

Comment:
Do we need this logic at all? we want user to see unread notification until he reopens the notification modal